### PR TITLE
Adding quantiles for ensemble outputs

### PR DESCRIPTION
**Pull request by anirban-chaudhuri**

### CODE REVIEW

File: src/pyciemss/Ensemble/interfaces.py [hunk 1]
```diff
@@ -102,11 +104,16 @@ def load_and_sample_petri_ensemble(
...
             - False do not output a visual
             - dict output a visual with the dictionary passed to the visualization as kwargs
+        alpha_qs: Optional[Iterable[float]]
+            - The quantiles required for estimating weighted interval score to test ensemble forecasting accuracy.
+        stacking_order: Optional[str]
+            - The stacking order requested for the ensemble quantiles to keep the selected quantity together for each state.
+            - Options: "timepoints" or "quantiles"
 
     Returns:
         samples:
```
Comment (SamWitty) [inline src/pyciemss/Ensemble/interfaces.py:114]: Can you split this out into two indented values. `samples` and `q_ensemble`, each with their own description as a list?

**Review by SamWitty — CHANGES_REQUESTED** [review]

File: src/pyciemss/Ensemble/interfaces.py (hunk 1 above)
Comment (anirban-chaudhuri) [reply src/pyciemss/Ensemble/interfaces.py:114]: This is the part I am not sure about whether this is the best option for outputs. Should both return statements be a dictionary instead?

Comment (anirban-chaudhuri) [reply src/pyciemss/Ensemble/interfaces.py:114]: Also the return type is not consistent

**Review by SamWitty — APPROVED** [review]
There's a bit of inconsistency with documentation of return types, but we can deal with that later.

**Pull request merged**